exonum-java-binding-core/rust/integration_tests/tests/jni_cache.rs

```diff
@@ -14,34 +14,23 @@
 
 extern crate integration_tests;
 extern crate java_bindings;
-#[macro_use]
-extern crate lazy_static;
 
 use integration_tests::vm::create_vm_for_tests_with_fake_classes;
-use java_bindings::{jni::JavaVM, utils::jni_cache, JniExecutor, MainExecutor};
+use java_bindings::utils::jni_cache;
 
 use std::{
     sync::{Arc, Barrier},
     thread::spawn,
 };
 
-lazy_static! {
-    pub static ref VM: Arc<JavaVM> = create_vm_for_tests_with_fake_classes();
-    pub static ref EXECUTOR: MainExecutor = MainExecutor::new(VM.clone());
-}
 
 #[test]
 // NOTE: This test is not supposed to reliably catch synchronization errors.
 fn concurrent_cache_read() {
```

Comment:
Will the test below always work? (cache_not_initialized)

Are these tests run in _separate_ processes?

Reply:
Well, it worked fine until now, but yes - probably it's a good idea to put it to a separate file. Tests are executed in the single process but in parallel in `RUST_TEST_THREADS` (or --test-threads) threads.